Scripts/GameManagement/GameManager.gd

```diff
@@ -49,3 +50,18 @@ func update_bullets():
 	for bullet in bullets_info:
 		world.get_node(bullet).global_position = bullets_info[bullet]['position']
 
+func player_died(player_name):
```

Comment:
I think this function should be named "delete_player" and function in World.tscn which now is named "delete_player" should be named "kill_player" maybe. "player_died" is not good name for function as it doesn't show what that function is doing. That name is more accurate for signals.